packages/eslint/README.md

```diff
@@ -38,51 +38,51 @@ export default {
 
 ## Options
 
-See more options here [eslint-config](http://eslint.org/docs/developer-guide/nodejs-api#cliengine).
+This plugin takes a configuration object intended for the [ESLint constructor](https://eslint.org/docs/developer-guide/nodejs-api#-new-eslintoptions) with the addition of a `throwOnWarning`, `throwOnError`, `formatter`, `include` and `exclude` prop.
 
 You can also use eslint configuration in the form of a `.eslintrc.*` file in your project's root. It will be loaded automatically.
 
-### fix
+### exclude
 
-Type: `Boolean`<br>
-Default: `false`
+Type: `string | string[]`<br>
+Default: `node_modules/**`
 
-If true, will auto fix source code.
+A single [`picomatch`](https://github.com/micromatch/picomatch) pattern or an array of patterns controlling which files this plugin should explicitly include. Gets forwarded to the [`createFilter`](https://github.com/rollup/plugins/tree/master/packages/pluginutils#createfilter) method of `@rollup/pluginutils`.
 
-### throwOnError
+### fix
 
-Type: `Boolean`<br>
+Type: `boolean`<br>
 Default: `false`
 
-If true, will throw an error if any errors were found.
+If true, will auto fix source code.
 
-### throwOnWarning
+### formatter
 
-Type: `Boolean`<br>
-Default: `false`
+Type: `function | string`<br>
+Default: `stylish`
 
-If true, will throw an error if any warnings were found.
+Custom error formatter, the name of a built-in formatter, or the path to a custom formatter.
 
 ### include
 
-Type: `Array | String`<br>
+Type: `string | string[]`<br>
```

Comment:
We do follow the TS casing/naming for option types. So `String` instead of `string`

Reply:
done